samples/Sample_10.avc.ref

```diff
@@ -20,7 +20,7 @@ UpdatePictureParameters
             "vui_parameters_present_flag": 1,
         },
         "StdVideoH264ProfileIdc": 100,
-        "StdVideoH264LevelIdc": 13,
+        "StdVideoH264LevelIdc": 3,
```

Comment:
Why does this value change ? With nvidia-parser the value is 13 and 3 is not a valid value right ?

Reply:
rigth, that's what I expiain

> > NVIDIA parser reports in StdVideoH264LevelIdc the extracte value from the bytestream, not the mapped data according the specification. This might a bug either in the parser or in vk_video_samples
> 
> I dont think they use LevelIdc in https://github.com/nvpro-samples/vk_video_samples . They just started to use Profile here [nvpro-samples/vk_video_samples@d82f6b2](https://github.com/nvpro-samples/vk_video_samples/commit/d82f6b270fccdfad553820307eb74ba4b1b53d98)

Right! they don't! That makes me think that is an actual bug both in vk_video_samples and the nvidia parser.